### DISCUSSION TITLE
How much computing power and how big of a dataset was used to train the official ControlNet models?

### DISCUSSION THREAD

**Discussion by SwayStar123** [lllyasviel/ControlNet#209] [General]
Also how was the dataset obtained. Mainly interested in depth and canny model but would be interesting to know the dataset for the scribble model

**Comment by geroldmeisinger** [comment]
most of it is explained in the original controlnet paper https://arxiv.org/abs/2302.05543 -> v1 and the readmes of the official github repos https://github.com/lllyasviel/ControlNet https://github.com/lllyasviel/ControlNet-v1-1-nightly

https://github.com/lllyasviel/ControlNet/issues/93#issuecomment-1436077011
> Given the current complicated situation outside research community, we refrain from disclosing more details about data. Nevertheless, researchers may take a look at that dataset project everyone know.
